src/test.js

```diff
@@ -356,8 +356,8 @@ Test.prototype = {
 	},
 
 	valid: function() {
-		var include,
-			filter = config.filter && config.filter.toLowerCase(),
+		var filter = config.filter,
+			regexFilter = /^!?\/([\w\W]*)\/(i?$)/.exec( filter ),
```

Comment:
Seems reasonable. If we include it on the method and add tests for it, I'm game.